conda_smithy/configure_feedstock.py

```diff
@@ -1741,9 +1741,13 @@ def _load_forge_config(forge_dir, exclusive_config_file, forge_yml=None):
         "build_with_mambabuild": True,
         # feedstock checkout git clone depth, None means keep default, 0 means no limit
         "clone_depth": None,
-        # Specific channel for package can be given with
-        # `${url or channel_alias}::package_name`, defaults to conda-forge channel_alias
         "remote_ci_setup": "conda-forge-ci-setup=3",
+        # Specific channel for packages for conda-forge-ci-setup,
+        # defaults to conda-forge
+        "remote_ci_setup_channels": ["conda-forge"],
+        # Extra packages that need installation or updating
+        # During the setup phase
+        "remote_ci_setup_extra_packages": [],
```

Comment:
I think we can change `remote_ci_setup` to be a list and there's no need for this new parameter.

Reply:
that is true.... but `remote_ci_setup` isn't a list. So it would make less than fun for backward compatibility. Does anybody use the old parameter? Can I break it and make it a list?

Reply:
You can use `conda_build.util.ensure_list` to make it a list.